linter/yaml_linter.json

```diff
@@ -0,0 +1,20 @@
+{
+    "problemMatcher": [
+        {
+            "owner": "yaml_linter_errors",
+            "severity": "error",
+            "pattern": [
+                {
+                    "regexp": "Error: (.*):(\\d+):(\\d+): (.*)",
```

Comment:
@ericLemanissier If you have a second, can you check my matcher? I cant figure out why it's not working 🙈 https://regex101.com/r/G5aaIa/1

Reply:
mmm, Maybe I'm late, but it seems to be properly picked now, right ? https://github.com/conan-io/conan-center-index/actions/runs/3300606265/jobs/5445276351#step:7:48

Reply:
Yeah I found someone else who figured it out ;)

Thanks for helping:)